Update troubleshooting java with SSLException handshake cipher suites (#252)
undefinedlabs/scope-docs: docs/java-troubleshooting.md

If you are using certain libraries or version libraries that are not officially supported by the Scope Java agent, you might have not see data in Scope after executing the test phase.

You can find further information in [Scope Java agent - Compatibility](java-compatibility.md) section.

> ## I'm getting `SSLException: Received close_notify during handshake` error:

Reviewer: I don't know if `I'm getting` adds much here

> ## I'm getting `SSLException: Received close_notify during handshake` error:

Reviewer 2: ```suggestion
## I'm getting `SSLException: Received close_notify during handshake` error
```

> This error may occur if you are using an outdated JDK 1.7 patch version due to some required cipher suites are not supported in that version.

Reviewer: the phrase is broken.
Either
```suggestion
This error may occur if you are using an outdated JDK 1.7 patch version because some required cipher suites are not supported in that version.
```
or
```suggestion
This error may occur if you are using an outdated JDK 1.7 patch version due to some required cipher suites that are not supported in that version.
```

I'd go for the first one

> To solve this issue, update your JDK 1.7 to `1.7.0_u131+` with the SSL handshake negotiation.

Reviewer 2: "with the SSL handshake negotiation" sounds weird. Is that a feature included in `1.7.0_u131+`?
```suggestion
To solve this issue, update your JDK 1.7 to `1.7.0_u131+` which comes with the required SSL cipher suites.
```

> Please, update your JDK 1.7 to `1.7.0_u131+` to solve this issue with the SSL handshake negotiation.

Reviewer: I'd go for 
```suggestion
To solve this issue, update your JDK 1.7 to `1.7.0_u131+` with the SSL handshake negotiation.
```
but that's just an opinion
